exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -0,0 +1,369 @@
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderInfo returns information on a current open order
+func (c *Coinbene) GetOrderInfo(orderID string) (exchange.OrderDetail, error) {
+	var resp exchange.OrderDetail
+	tempResp, err := c.FetchOrderInfo(orderID)
+	if err != nil {
+		return resp, err
+	}
+	resp.Exchange = c.Name
+	resp.ID = orderID
+	resp.CurrencyPair = currency.NewPairWithDelimiter(tempResp.Order.BaseAsset, "/", tempResp.Order.QuoteAsset)
+	timestamp := tempResp.Order.OrderTime
+	resp.OrderDate = time.Unix(timestamp, 0)
+	resp.ExecutedAmount = tempResp.Order.FilledAmount
+	resp.Fee = tempResp.Order.TotalFee
+	return resp, nil
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (c *Coinbene) GetDepositAddress(cryptocurrency currency.Code, accountID string) (string, error) {
+	return "", common.ErrFunctionNotSupported
+}
+
+// WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is
+// submitted
+func (c *Coinbene) WithdrawCryptocurrencyFunds(withdrawRequest *exchange.WithdrawRequest) (string, error) {
+	return "", common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is
+// submitted
+func (c *Coinbene) WithdrawFiatFunds(withdrawRequest *exchange.WithdrawRequest) (string, error) {
+	return "", common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is
+// submitted
+func (c *Coinbene) WithdrawFiatFundsToInternationalBank(withdrawRequest *exchange.WithdrawRequest) (string, error) {
+	return "", common.ErrFunctionNotSupported
+}
+
+// GetWebsocket returns a pointer to the exchange websocket
+func (c *Coinbene) GetWebsocket() (*wshandler.Websocket, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (c *Coinbene) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	var tempResp exchange.OrderDetail
+	var tempData OpenOrderResponse
+	if len(getOrdersRequest.Currencies) == 0 {
+		allPairs, err := c.GetAllPairs()
+		if err != nil {
+			return resp, err
+		}
+		for a := range allPairs.Data {
+			getOrdersRequest.Currencies = append(getOrdersRequest.Currencies, currency.NewPairFromString(allPairs.Data[a].Symbol))
+		}
+	}
+	var err error
+	for x := range getOrdersRequest.Currencies {
+		tempData, err = c.FetchOpenOrders(exchange.FormatExchangeCurrency(c.Name, getOrdersRequest.Currencies[x]).String())
+		if err != nil {
+			return resp, err
+		}
+		for y := range tempData.OpenOrders {
+			tempResp.Exchange = c.Name
+			tempResp.CurrencyPair = getOrdersRequest.Currencies[x]
+			if tempData.OpenOrders[y].OrderType == buy {
+				tempResp.OrderSide = exchange.BuyOrderSide
+			}
+			if tempData.OpenOrders[y].OrderType == sell {
+				tempResp.OrderSide = exchange.SellOrderSide
+			}
+			tempResp.OrderDate = time.Unix(tempData.OpenOrders[y].OrderTime, 0)
```

Comment:
According to the docs fetching open orderids is returned like ` "orderTime": "2019-06-12T09:39:27.0Z"` and you don't do any conversion in the API implementation. This will die 🥀 🌤   You'll need to update your struct to use a string and then use `time.Parse(stringerino)` instead